front/src/components/surveyExport.js

```diff
@@ -0,0 +1,118 @@
+import axios from 'axios'
+import React, {Component} from "react";
+import {Button} from "reactstrap";
+import {Link} from "react-router-dom";
+
+class SurveyExport extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            survey: -1,
+            frontUrl: "http://localhost:3000",
+            serverUrl: "http://localhost:8080"
+        }
+    }
+
+    componentDidMount() {
+        let survey_id = window.location.href
+        survey_id = survey_id.replace(this.state.frontUrl + "/surveyexport/", "")
+        axios.get(this.state.serverUrl + "/surveys/" + survey_id)
+            .then((response) => {
+                this.setState({survey: response.data})
+            })
```

Comment:
- Please add `{headers: {authorization: AuthenticationService.getAuthToken()}}` to all your axios methods - right now all methods return error 401. 
- Catching errors is missing (in all axios methods)